src/test/java/com/google/api/codegen/testdata/python_doc_message_library.baseline

```diff
@@ -623,7 +623,7 @@ class Book(object):
       author (string): The name of the book author.
       title (string): The title of the book.
       read (bool): Value indicating whether the book has been read.
-      rating (:class:`google.example.library.v1.library_pb2.Book.Rating`): For testing enums.
+      rating (enum from :class:`google.example.library.v1.enums.Rating`): For testing enums.
```

Comment:
Done.

Thanks for the link! I was actually trying to figure out what the nomenclature was when I wrote "enum from" -- I was confused by the fact that an "enum member" is not exactly an "instance" of an enum. Good to know what the precedent is here.